File: app/src/main/java/com/example/numad25sp_wenyupan_java/AboutMeActivity.java
package com.example.numad25sp_wenyupan_java;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.widget.TextView;

public class AboutMeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about_me);

        // Find TextView and set name & email
        TextView aboutMeText = findViewById(R.id.aboutMeText);
        aboutMeText.setText("Wenyu Pan\npan.we@northeastern.edu");
    }
}

File: app/src/main/java/com/example/numad25sp_wenyupan/AboutMeActivity.kt
package com.example.numad25sp_wenyupan

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.*
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.ui.unit.dp
import com.example.numad25sp_wenyupan.ui.theme.NUMAD25Sp_WenyuPanTheme

class AboutMeActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            NUMAD25Sp_WenyuPanTheme {
                AboutMeScreen()
            }
        }
    }
}

@Composable
fun AboutMeScreen() {
    Column(
        modifier = Modifier.fillMaxSize(),
        horizontalAlignment = Alignment.CenterHorizontally,
        verticalArrangement = Arrangement.Center
    ) {
        Text(text = "Wenyu Pan", modifier = Modifier.padding(8.dp))
        Text(text = "pan.we@northeastern.edu", modifier = Modifier.padding(8.dp))
    }
}

@Preview(showBackground = true)
@Composable
fun AboutMePreview() {
    NUMAD25Sp_WenyuPanTheme {
        AboutMeScreen()
    }
}
